src/main/java/de/unistuttgart/informatik/fius/icge/simulation/Simulation.java

```diff
@@ -30,6 +28,7 @@
     private int _tickCount = 0;
     private TimerTask _timerTask;
     private Semaphore _timerTaskSem;
+    private ArrayList<Entity> _entities = new ArrayList<>();

```

Comment:
The `Simulation` class does already have a `HashMap` called `_entityObjects` which maps `Entity`s to the associated `WorldObject` representing the current state of that `Entity`. Why add another list?

Reply:
Also: Why is this included in this pull request regarding the speed slider?

Reply:
Thanks for the information. Haven't noticed it while scrolling through the code...